### ISSUE TITLE
ignore if this is already fixed...

### ISSUE THREAD

**Issue by publicsphere** [smrose/ps#19]
currently (12:37 monday) if I change any PGHCI pattern assessments, they don't show up in subsequent assessment pages or "results" from the menu at the top rigtht. I.e. the changes don't seem to be recorded.

**Comment by smrose** [comment]
Please don't use a title like ignore "if this is already fixed" for issues. When I got to look at the list of issues to decide which to work on, I'm looking at the titles to understand and prioritize, and that title tells me nothing about the issue.

**Comment by smrose** [comment]
The expectation is that each project will have one or more teams assigned, and that those teams will have one or more members. It's true that assessments can be made by a superuser regardless they are a member of any team that is participating in a project, but that's probably a mistake to allow because we end up with anomalous situations such as zero users being listed in the Users section of the results page. I have therefore assigned the team "Post growth" to participate in the PGHCI project. That team has two members: "rose" and "doug".

To test the claim that results aren't being recorded, I edited my own assessment and ranked both patterns in the "Framework / Ontology" language as excellent, then observed that the assessments were recorded by looking at the results page. Then I reassessed them - my earlier assessments were properly shown in the assessments page, evidence that they recorded - both as good and looked again at the results page. There, I saw that the counts for those patterns had changed appropriately.

In short, I can't reproduce this. However, all my tests were performed after I assigned a team to the project, whereas all of yours were before. I'm going to close this issue and create another to ensure that superusers can't enter assessments, as it seems to be screwing the pooch on multiple levels.

**Comment by publicsphere** [comment]
this still doesn't work for me. I tried it with "older" patterns and it didn't work (screen shots attached). When I tried it with new patterns such as those in the "Framework / Ontology"  language, it DID work!  Note that I had only changed the last four pattern assessment.

<img width="1680" alt="Screenshot 2024-05-06 at 3 10 06 PM" src="https://github.com/smrose/ps/assets/6635482/52f5d49f-1d62-416b-a86d-151a2121d5c6">
<img width="1680" alt="Screenshot 2024-05-06 at 3 10 46 PM" src="https://github.com/smrose/ps/assets/6635482/d5fd61d9-1828-491e-a248-d700f96b1e63">
